pkg/i18n/dutch.go

```diff
@@ -214,6 +214,9 @@ func addDutch(i18nObject *i18n.Bundle) error {
 		}, &i18n.Message{
 			ID:    "NoTrackingThisBranch",
 			Other: "deze branch wordt niet gevolgd",
+		}, &i18n.Message{
+			ID:    "CommitMessage",
+			Other: "{{.keyBindClose}}: Sluiten, {{.keyBindNewLine}}: new line, {{.keyBindConfirm}}: Bevestigen",
```

Comment:
is `neue textzeile` the translation for `new line`?

Reply:
Nope: "Nieuwe lijn"
Can you also change: "Bevestigen" to "Bevestig"

Reply:
the `CloseConfirm` message use: `Bevestigen`. Should I change this too?

Reply:
Yes if you want to :)